### ISSUE TITLE
Revise the `user-config.yml` to support updated MO user config requirements

### ISSUE THREAD

**Issue by chrisbillowsMO** [ESMValGroup/ESMValCore#2628]
MO users now need two alternative ESMValTool configurations.

The proposed solution is to include both specifications in the current `user-config.yml`.

I am working on this issue.

**Comment by rswamina** [comment]
@chrisbillowsMO  - Does this include the location of the CMOR-ized TerraFIRMA overshoot data (simulations)  on JASMIN? 

The location is : `gws/nopw/j04/terrafirma/MOHC/UKESM1-2/`

Thanks!

**Comment by sloosvel** [comment]
Hi @chrisbillowsMO  ,

The code freeze for version 2.12 is scheduled for next week on Feb 11th. Do you think you will have time to address this issue for this release? If not, let me know and I will move it to the next milestone.

**Comment by chrisbillowsMO** [comment]
> Hi [@chrisbillowsMO](https://github.com/chrisbillowsMO) ,
> 
> The code freeze for version 2.12 is scheduled for next week on Feb 11th. Do you think you will have time to address this issue for this release? If not, let me know and I will move it to the next milestone.

Hi @sloosvel - thanks for the message, I've moved the issue to the next milestone.

**Comment by ehogan** [comment]
Hi @sloosvel 👋 I just discussed this with @chrisbillowsMO and we would really like this in the next release. However, we won't be able to create a PR until Tuesday morning. Would that be ok? 🤞

**Comment by sloosvel** [comment]
Do you think you can have it merged by Tuesday afternoon? If so, it should be ok.

**Comment by ehogan** [comment]
> Do you think you can have it merged by Tuesday afternoon? If so, it should be ok.

Yes! @chrisbillowsMO will work on the PR, I will test on Tuesday morning, @alistairsellar will review, and I will merge in the afternoon 💪

**Comment by ehogan** [comment]
> [@chrisbillowsMO](https://github.com/chrisbillowsMO) - Does this include the location of the CMOR-ized TerraFIRMA overshoot data (simulations) on JASMIN?
> 
> The location is : `gws/nopw/j04/terrafirma/MOHC/UKESM1-2/`
> 
> Thanks!

Hi @rswamina 👋 This PR is just for changes related to MO paths in the user configuration file. If changes are required to JASMIN paths, would it be possible for you to open a separate issue, please? 😊

**Comment by rswamina** [comment]
That's fine @ehogan . I have an issue open already. Thanks!
